src/main/java/springweb/courseproject/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,23 @@
 package springweb.courseproject.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import springweb.courseproject.validation.Author;
+import springweb.courseproject.validation.Isbn;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @Author

```

Comment:
Don't need to create custom validation for author, it's redundant
```suggestion
    @NotBlank
```